packages/log-structured/package.json

```diff
@@ -0,0 +1,21 @@
+{
+    "name": "@idearium/log-structured",
+    "version": "1.0.0",
+    "description": "Idearium logger structured logging formatter",
+    "repository": "https://github.com/idearium/idearium-lib",
+    "author": "Scott Mebberson <scott@idearium.io>",
+    "license": "MIT",
+    "bin": {
+        "structured": "bin/structured.js"
+    },
+    "scripts": {
+        "test": "jest"
+    },
+    "dependencies": {
+        "pump": "3.0.0"
+    },
+    "devDependencies": {
+        "pino-http": "5.5.0",
+        "sink": "0.1.1"
```

Comment:
Is this right? Shouldn't they just be normal dependencies?

Reply:
@allanchau, that only used in the tests...

Reply:
@smebberson Oh sorry wrong file, I was suppose to make the comment in packages/log-http/package.json. `pino-http` at least should be a dependency?